src/components/Book.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ]  @MasumaJaffery It's not good practice disabling a linter check in your file, linter checks help us write quality code, kindly consider removing this line